examples/nbody/nbody.cpp

```diff
@@ -276,6 +276,9 @@ struct UpdateKernel
                 pos2 += threads
             )
                 temp( pos2 + threadBlockIndex ) = particles( start2 + pos2 + threadBlockIndex );
```

Comment:
while we are on it: this line is ill-aligned :)
(four less spaces pls)

Reply:
why? its the statement of the for loop

Reply:
oh right, hard to see!

Reply:
thats true, maybe we should use `{}` here:

```diff
-                pos2 += threads
-            )
+                pos2 += threads )
+            {
                 temp( pos2 + threadBlockIndex ) = particles( start2 + pos2 + threadBlockIndex );
+            }
```